libs/application/templates/document-provider-onboarding/src/forms/DocumentProviderApplication.ts

```diff
@@ -6,28 +6,54 @@ import {
   buildSubmitField,
   buildCustomField,
   buildDividerField,
+  buildDescriptionField,
+  buildCheckboxField,
+  buildLinkField,
 } from '@island.is/application/core'
 import { Form, ApplicationTypes, FormModes } from '@island.is/application/types'
-import { m } from './messages'
+import { m } from '../lib/messages'
 
 export const DocumentProviderOnboarding: Form = buildForm({
   id: ApplicationTypes.DOCUMENT_PROVIDER_ONBOARDING,
   title: m.formName,
   mode: FormModes.DRAFT,
+  renderLastScreenButton: true,
   children: [
     buildSection({
       id: 'termsOfAgreement',
       title: m.termsSection,
       children: [
-        buildCustomField(
-          {
-            id: 'termsOfAgreement',
-            title: m.termsTitle,
-            description: m.termsSubTitle,
-            component: 'TermsOfAgreement',
-          },
-          {},
-        ),
+        buildMultiField({
+          id: 'termsOfAgreement',
+          title: m.termsTitle,
+          children: [
+            buildCustomField(
+              {
+                id: 'thankYouScreen',
+                title: m.thankYouScreenTitle,
+                component: 'ThankYouScreen',
+              },
+              {},
```

Comment:
Did you remove the `ThankYouScreen` custom component? 🤔

Reply:
Well spotted, I'll check this out